### ISSUE TITLE
Tibet research

### ISSUE THREAD

**Issue by IhateTrains** [loup99/WtWSMS#397]
UN document: http://unesdoc.unesco.org/images/0010/001046/104612e.pdf#xml=http://www.unesco.org/ulis/cgi-bin/ulis.pl?database=&set=4DC27DB7_0_203&hits_rec=6&hits_lng=eng

**Comment by IhateTrains** [comment]
![image](https://user-images.githubusercontent.com/29546927/46908709-2cf92600-cf27-11e8-89d6-6863853e72e5.png)
Here's how Tibet looks in the 482 mod from the Workshop. Note that it's unlikely to be accurate, but at least the realm names could help someone in research.

**Comment by IhateTrains** [comment]
Realms that should be portrayed (I will try to keep updating it):
* Zhangzhung (https://en.wikipedia.org/wiki/Zhangzhung)

**Comment by IhateTrains** [comment]
**Religion**: Bön, Tibetan Buddhism is probably out of the timeframe (https://en.wikipedia.org/wiki/History_of_Tibetan_Buddhism#First_dissemination_(7th%E2%80%939th_centuries))

**Comment by tannerflick4** [comment]
Yeah Buddhism really only came into Tibet with the later kings (and emperors) of Tibet. https://www.metmuseum.org/toah/ht/06/ssh.html

**Comment by loup99** [comment]
I have access @tannerflick4

**Comment by IhateTrains** [comment]
some info from the forum, has problems with transliteration consistency:
https://forum.paradoxplaza.com/forum/index.php?threads/summa-gentium-locorumque.443104/page-6#post-10698536

A map that shows a possible location of the tribes and Zhanzhung:
![image](https://user-images.githubusercontent.com/29546927/47956720-3cf5aa00-dfa9-11e8-8fb9-a7c39be34c1e.png)

**Comment by IhateTrains** [comment]
Info on specific Tibetan clans:

- Dro clan: an important Tibetan family based in Kokonor (modern-day Qinghai)

**Comment by loup99** [comment]
While Nepal isn't a part of Tibet, it is adjacent and will be added together with it. The Licchavi kingdom should hold the area throughout the 476-700 period. On Wikipedia there is a list of rulers: https://en.wikipedia.org/wiki/Licchavi_(kingdom)
